fiftyone/zoo/models/manifest-tf.json

```diff
@@ -1237,6 +1237,41 @@
             },
             "tags": ["detection", "coco", "tf1"],
             "date_added": "2020-12-11 13:45:51"
+        },
+        {
```

Comment:
@ehofesmann looks like you lost the other package updates that you originally had here in the merge

Reply:
What other package updates do you mean?

Reply:
The diffs you see in this commit https://github.com/voxel51/fiftyone/pull/740/commits/46f5eb3fc9503d1248e1cff73aed60c84dc1f0b5 no longer appear now

Reply:
Gotcha